openedx/core/lib/xblock_builtin/xblock_discussion/utils.py

```diff
@@ -0,0 +1,17 @@
+"""
+Utils for DiscussionXBlock
```

Comment:
Hmm, this is tricky. I agree that pipeline_js seems like a reasonable home, except that it is xmodule-centric and CSS methods don't fit naturally. The only other place that uses ``PIPELINE_ENABLED`` is ``openedx/core/lib/xblock_utils.py`` which is at least not discussion specific. Maybe put them there and at some point in the future we could have a cleaner pipeline utils location.